boards/esp32c3/kitten_grapebit_c3.board.json

```diff
@@ -0,0 +1,61 @@
+{
+    "$schema": "../esp32deviceconfig.schema.json",
+    "devName": "Kittenbot Grape:Bit ESP32-C3 DeviceScript",
+    "productId": "0x3635315b",
+    "$description": "Stemcraft C3",
+    "jacdac": {
+        "$connector": "Jacdac",
+        "pin": 5
+    },
+    "led": {
+        "pin": 8,
+        "activeHigh": true
+    },
+    "i2c": {
+        "$connector": "Qwiic",
```

Comment:
Is there a JST-SH-4 (Qwiic/STEMMA QT) connector on the board? If not, please remove the $connector entry.

Reply:
Yeah, we are using some imu sensor not in the lib now. We will make another PR to jacdac-c.

Reply:
For DeviceScript, we switched to I2C drivers written in TypeScript like this https://github.com/microsoft/devicescript/blob/main/packages/drivers/src/ltr390.ts

We do not yet have a driver for IMU though. I can try to add something.

Reply:
I manage to read the on board IMU with the following code snippet, but don't have any idea on how to subscribe with continues reading. https://gist.github.com/xmeow/7ea077bf737ac6eddbbdf132a6c74d41